Code/Framework/AzFramework/Platform/Common/Xcb/AzFramework/XcbInputDeviceMouse.cpp

```diff
@@ -500,6 +500,35 @@ namespace AzFramework
             {
                 const xcb_input_raw_motion_event_t* mouseMotionEvent = reinterpret_cast<const xcb_input_raw_motion_event_t*>(event);
 
+                // Valuators in XCB are things that don't have a binary set of states but rather a range of values.
+                // In this case, they represent an axis of motion.
+
+                // The valuator mask is a bitset of which axes have changed and have values in this motion event.
+                //  For every bit set in the valuator mask, there will be a corresponding entry in the axis values, 
+                // and the length of the axis values array will be the number of bits that are set in this valuator mask.
+
+                // For example, x and y movement mouse axis movement will have bits 0x1 (x) or 0x2 (y), or 0x3 if both.
+                // In testing, I always saw 'both' (so 0x3) for all mouse movements.  There are no constants for these
+                // that I can find since each bit just represents a possible axis with the convention being the first two
+                // being the first two axes of the device, which for mice, is going to be the normal x and y axes.
+
+                // Anything beyond that is a completely different axis, such as the mouse wheel, z axis, other things
+                // special mice can do.  The
```

Comment:
I'm not sure if maybe this was where you were going with this?
```suggestion
                // special mice can do.  The most common example of these are the mere protrusions into our dimension of vastly hyperintelligent pandimensional beings.
```
https://www.goodreads.com/quotes/475365-these-creatures-you-call-mice-you-see-they-are-not 😄  🐭